Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: git@github.com:wordpress-mobile/release-toolkit.git
-  revision: c78ca386d4efbe7b46ba9c09a4e45ba584b21835
+  revision: 262e6800830abc75c7d9b6817537b4c099588634
```

Comment:
This is pending a release in https://github.com/wordpress-mobile/release-toolkit/pull/283

I'm ok to merge this pointing to a temp branch now and switch back to pointing at `1.3.1` once the toolkit PR get approved and merged later and `1.3.1` gets released.